OpenRA.Mods.Common/Traits/Render/Hovers.cs

```diff
@@ -19,35 +19,71 @@
 namespace OpenRA.Mods.Common.Traits.Render
 {
 	[Desc("Changes the visual Z position periodically.")]
-	public class HoversInfo : ConditionalTraitInfo, Requires<IMoveInfo>
+	public class HoversInfo : PausableConditionalTraitInfo, Requires<IMoveInfo>
 	{
 		[Desc("Amount of Z axis changes in world units.")]
-		public readonly int OffsetModifier = -43;
+		public readonly int OffsetModifier = 64;
```

Comment:
Can you elaborate on why this changed?

We generally require an update rule when we change default values so that we don't unexpectedly change behaviour for mods.

Reply:
Oh, that was changed as I somehow tried to match TS hover mrls look, can revert that

Reply:
Yes, please revert this (and add the needed yaml to the hover mrls instead).